appinfo/info.xml

```diff
@@ -17,8 +17,8 @@
 	<screenshot>https://raw.githubusercontent.com/nextcloud/twofactor_totp/dd1e48deec73a250886f35f3924186f5357f4c5f/screenshots/enter_challenge.png</screenshot>
 	<screenshot>https://raw.githubusercontent.com/nextcloud/twofactor_totp/dd1e48deec73a250886f35f3924186f5357f4c5f/screenshots/settings.png</screenshot>
 	<dependencies>
-		<php min-version="7.0" max-version="7.3" />
-		<nextcloud min-version="15" max-version="16" />
+		<php min-version="7.1" max-version="7.3" />
+		<nextcloud min-version="16" max-version="16" />
```

Comment:
could we make it php7.1+ but nc15+ for now? I don't want to drop the 15 support on master yet